docs/de/docs/deployment/index.md

```diff
@@ -0,0 +1,7 @@
+# Deployment - Einführung
+
+Das Bereitstellen einer **FastAPI** Anwendung ist relativ einfach.
+
+Hierbei gibt es abhängig von der Situation und den eingesetzen Technologien mehrere Möglichkeiten.
+
+In den nächsten Kapiteln befinden sich hierzu mehrere nähere Informationen und einige der möglichen Technologien werden vorgestellt.
```

Comment:
Alternative suggestion: 
```In den nächsten Abschnitten erfahren Sie mehr über die Details, die Sie beachten sollten, sowie über einige der Techniken, die Sie dabei anwenden können.```

Reply:
This seems to be better, but i would say: `In den nächsten Abschnitten erfahren Sie mehr über wichtige Details, sowie über einige der Techniken, die Sie dabei anwenden können.`